draft-ietf-quic-transport.md

```diff
@@ -2116,9 +2116,13 @@ packets.
 A server conveys a preferred address by including the preferred_address
 transport parameter in the TLS handshake.
 
-Once the handshake is finished, the client SHOULD initiate path validation (see
-{{migrate-validate}}) of the server's preferred address using the connection ID
-provided in the preferred_address transport parameter.
+Servers MAY communicate a preferred address of each address family (IPv4 and
+IPv6) to allow clients to pick the one most suited to their network attachment.
```

Comment:
After all our discussion, the question I have here is whether this preferred address might be used for a migating client.  Say I use v6 now, but then migrate to a network that doesn't have v6.  Is it acceptable for me to use the v6 preferred address during that migration?  The text here is silent on that point.

Reply:
@martinthomson I'm thinking we may want more deployment experience before we decide if we want to recommend or prohibit that.

Reply:
I'm only asking if it is *allowed*.

Reply:
It is currently allowed - there is no text prohibiting it.

Reply:
We may want to explicitly state this. See my comment above.

Reply:
Good point, I've added some text to the _Interaction of Client Migration and Preferred Address_ section.